tests/integration/helpers.py

```diff
@@ -333,6 +333,30 @@ def kubernetes_test_pipeline_script() -> str:
     )
 
 
+def declarative_pipeline_script() -> str:
```

Comment:
I wonder if this should be a fixture instead

Reply:
I think I've had a discussion about this with someone, and it was decided that these should be functions instead of fixtures because we have a lot of them and making them a fixture would makes the `conftest.py` quite big and makes it quite complicated to track

Reply:
Works for me